components/toggle/Toggle.tsx

```diff
@@ -0,0 +1,51 @@
+import styles from "./Toggle.module.css";
+
+type ToggleProps = {
+  showMap: boolean;
+  setShowMap: Function;
+};
+
+const Toggle: React.FC<ToggleProps> = ({ showMap, setShowMap }) => {
```

Comment:
In the future we might use this toggle for other things, in the case we could name `showMap` and `setShowMap` to: `state` and `setState` for example